doc/puppetWinSW.md

```diff
@@ -0,0 +1,9 @@
+## Puppet Module Now Available!
```

Comment:
It's not a blog post. So I would refactor the text to be a kind of the documentation page